tests/flytekit/unit/types/file/test_image.py

```diff
@@ -0,0 +1,22 @@
+import PIL.Image
+
+from flytekit import task, workflow
+
+
+@task(disable_deck=False)
```

Comment:
Nit: can you use enable_deck?

Reply:
I can't find any other place using `PIL.Image` in flytkeit